.changeset/loud-dodos-crash.md

```diff
@@ -0,0 +1,10 @@
+---
+"@aws-amplify/ui-react-liveness": patch
+"@aws-amplify/ui-react": patch
```

Comment:
Think this package can be removed here

Reply:
Think it should be there since the UI package was updated?

Reply:
I think ui-react will get auto bumped with one of those "Updated dependencies" notes in the changelog since ui is bumped, so it doesn't need to explicitly be added here if it wasn't touched in the changed files.